### ISSUE TITLE
@summary function is broken with non-numeric columns

### ISSUE THREAD

**Issue by tk3369** [TidierOrg/TidierData.jl#122]
The `@summary` function throws an exception if the data frame has any non-numeric columns. I think it should just exclude them.
```julia
julia> df = DataFrame(name = ["a","b"], val = [1,2])
2×2 DataFrame
 Row │ name    val   
     │ String  Int64 
─────┼───────────────
   1 │ a           1
   2 │ b           2

julia> @summary(df)
ERROR: MethodError: no method matching isfinite(::String)
The function `isfinite` exists, but no method is defined for this combination of argument types.

Closest candidates are:
  isfinite(::Missing)
   @ Base missing.jl:101
  isfinite(::BigFloat)
   @ Base mpfr.jl:1048
  isfinite(::Complex)
   @ Base complex.jl:150
  ...

Stacktrace:
 [1] #_quantile#57
   @ ~/.julia/packages/Statistics/gbcbG/src/Statistics.jl:1029 [inlined]
 [2] _quantile
   @ ~/.julia/packages/Statistics/gbcbG/src/Statistics.jl:1004 [inlined]
 [3] quantile!(v::Vector{String}, p::Float64; sorted::Bool, alpha::Float64, beta::Float64)
   @ Statistics ~/.julia/packages/Statistics/gbcbG/src/Statistics.jl:980
 [4] quantile(v::Vector{String}, p::Float64; sorted::Bool, alpha::Float64, beta::Float64)
   @ Statistics ~/.julia/packages/Statistics/gbcbG/src/Statistics.jl:1093
 [5] summary_stats(df::DataFrame)
   @ TidierData ~/.julia/packages/TidierData/C04bk/src/summary.jl:7
 [6] macro expansion
   @ ~/.julia/packages/TidierData/C04bk/src/summary.jl:29 [inlined]
 [7] top-level scope
   @ REPL[111]:1
```

**Comment by drizk1** [comment]
I'll fix this soon and see if theres any value to leverage the simple string info from `DataFrames.describe` in this function. 

In the mean time the following will allow you to avoid the error 

```
@chain df @select(where(!is_string)) @summary()
```

**Comment by kdpsingh** [comment]
Resolved in #124.
